Robot_Adapter/CRUD/Create/Elements/FEMeshes.cs

```diff
@@ -39,7 +39,6 @@ public partial class RobotAdapter
 
         private bool CreateCollection(IEnumerable<FEMesh> fEMeshes)
         {
-            int fMeshFaceIdx = m_RobotApplication.Project.Structure.FiniteElems.FreeNumber;
```

Comment:
Dont think this change should be made. Faces are not handeled separately by the adapter (and can not really be due to the structure of the classes). Would leave the create for FEMeshes unchanged